lang/java/reef-runtime-multi/src/main/avro/RuntimeDefinition.avsc

```diff
@@ -0,0 +1,34 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+ [
+/*
+ * Defines the schema for runtime definition. This avro object is used to pass runtimes definitions
+ * to the runtimes host
+ */
+ {
+    "namespace":"org.apache.reef.runtime.multi.utils",
+    "type":"record",
+    "name":"RuntimeDefinition",
+    "fields":[
+      {"name":"runtimeName","type":"string"},
+      {"name":"serializedConfiguration","type":"string"},
+      {"name":"defaultConfiguration","type":"boolean"}
```

Comment:
What does this do? Can you add some documentation by using the `"doc"` attribute?